docs/docs/how-to/local-development/javascript-tooling.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Javascript tooling
+---
+
+Gatsby offers support for the standard Javascript toolchain.
+
+## Typescript
+
+Gatsby supports Typescript automatically, with the [ability to modify the default configuration](gatsby-plugin-typescript/) if desired.
+
+## Bundling and transpilation
+
+Gatsby uses webpack to bundle files and Babel to transpile Javascript.
+
+Gatsby ships with a default configuration that should work well for most projects, but if you want to modify it to load dependencies differently, use custom relative paths, include specific polyfills, or other customizations, you can [add custom webpack config](/docs/how-to/custom-configuration/add-custom-webpack-config/) and/or [use a custom .babelrc](/docs/how-to/custom-configuration/babel/)
```

Comment:
mention that there are many gatsby plugins which handle webpack changes — often it's easier to use a community-support webpack modification than maintaining your own